src/qibo/models/utils.py

```diff
@@ -214,3 +214,46 @@ def vqe_loss(params, circuit, hamiltonian):
     result = hamiltonian.backend.execute_circuit(circuit)
     final_state = result.state()
     return hamiltonian.expectation(final_state)
+
+
+def matrix_to_pauli_operators(matrix):
+    """Calculates the Pauli operators for a given matrix.
+    Code created as part of QuantumSpain Project (https://quantumspain-project.es/)
```

Comment:
Here and below: please, avoid explicit mentions in the docstrings. The code accepted for Qibo belongs to the project, according to the specified license. There is no further split (take into account that it will be maintained by other people as well, so the lines will progressively blur, and there is not much point in making fine-grained distinctions - moreover, Git knows exactly who contributed the code).

Instead, I'd propose to explicitly mention Quantum Spain among the other supporting institutions in the [README](https://github.com/qiboteam/qibo/?tab=readme-ov-file#supporters-and-collaborators) and the [docs](https://qibo.science/qibo/stable/index.html#supporters-and-collaborators).

Reply:
Thanks! I have romove the credit line from the code and added onto README and doc